festim/hydrogen_transport_problem.py

```diff
@@ -431,3 +523,13 @@ def iterate(
 
         # update previous solution
         self.u_n.x.array[:] = self.u.x.array[:]
+
+    def update_time_dependent_values(self, t):
```

Comment:
Why passing `t` as an argument since it's an attribute of the class?

Reply:
It was to clean the script a bit, seeing as the time will be used in all the updating variables, but can just use the attribute otherwise

Reply:
`float(self.t)` yes, I can change it